### PR TITLE
DM-28378: Added the IDL Conda package job to the shared library. Renamed the b…

### PR DESCRIPTION
**Pull request by rbovill**
…uild_conda function to build_csc_conda, for clarity.

Added .swp files to .gitignore.

Using the build parameters in the trigger SalObj job stage.

### CODE REVIEW

File: src/org/lsst/ts/jenkins/components/Csc.groovy [hunk 1]
```diff
@@ -33,6 +33,46 @@ def build_conda(label) {
...
+    sh """
+        # yum clean all ; yum makecache fast; yum update ; 
+        yum install -y --enablerepo=${rpm_repo} ts_sal_runtime-${params.xml_version}-${params.sal_version}.el7.x86_64
+        cd ${HOME}/conda
+        source /home/saluser/miniconda3/bin/activate 
+        conda config --add channels conda-forge
+        conda config --add channels lsstts
+        source ${OSPL_HOME}/release.com
+        conda build -c lsstts/label/${label} --prefix-length 100 .
+        ls /home/saluser/miniconda3/conda-bld/linux-64
```
Comment (couger01) [inline src/org/lsst/ts/jenkins/components/Csc.groovy]: Do you need an `ls` command at the end of this function here.

**Review by couger01 — CHANGES_REQUESTED** [review]
I only have one question that could be addressed otherwise looks great! Good job!

File: src/org/lsst/ts/jenkins/components/Csc.groovy (hunk 1 above)
Comment (rbovill) [reply src/org/lsst/ts/jenkins/components/Csc.groovy]: I removed the left-over debugging statement.

**Review by couger01 — APPROVED** [review]
LGTM.

**Pull request merged**